### PR TITLE
Deploy Snaxchain to testnet

### PR DESCRIPTION
**Pull request by bruce-riley**
The contracts are available here:

- [Core](https://explorer-snaxchain-s50q0kjngn.t.conduit.xyz/address/0xBB73cB66C26740F31d1FabDC6b7A46a038A300dd)
- [TokenBridge](https://explorer-snaxchain-s50q0kjngn.t.conduit.xyz/address/0xa10f2eF61dE1f19f586ab8B6F2EbA89bACE63F7a)

### CODE REVIEW

File: sdk/js/src/utils/consts.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ export const CHAINS = {
   linea: 38,
   berachain: 39,
   seievm: 40,
+  snaxchain: 43,
```
Comment (panoel) [inline sdk/js/src/utils/consts.ts:41]: We are still updating the old sdk?  Is there some coupling to sdkV2 to be had?

Comment (panoel) [reply sdk/js/src/utils/consts.ts:41]: Maybe someone, ahem, you, could take out a PR in the wormhole-sdk-ts repo?

**Review by panoel — APPROVED** [review]
I didn't check the token bridge emitter address.

File: sdk/testnet_consts.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ var knownTestnetTokenbridgeEmitters = map[vaa.ChainID]string{
 	vaa.ChainIDXLayer:          "000000000000000000000000dA91a06299BBF302091B053c6B9EF86Eff0f930D",
 	vaa.ChainIDLinea:           "000000000000000000000000C7A204bDBFe983FCD8d8E61D02b475D4073fF97e",
 	vaa.ChainIDBerachain:       "000000000000000000000000a10f2eF61dE1f19f586ab8B6F2EbA89bACE63F7a",
+	vaa.ChainIDSnaxchain:       "000000000000000000000000a10f2eF61dE1f19f586ab8B6F2EbA89bACE63F7a",
```
Comment (johnsaigle) [inline sdk/testnet_consts.go:42]: Should this have the same value as Berchain?

Comment (bruce-riley) [reply sdk/testnet_consts.go:42]: Yup. Some times the contracts get deployed to the same address on different chains. If you see the link for the token bridge up in the PR description, you'll see that's the token bridge.

Comment (johnsaigle) [reply sdk/testnet_consts.go:42]: Ah, thanks. I saw that the contract address matched what you posted but I just wanted to double-check that a duplicate value is safe. 👍🏻

File: ethereum/env/.env.snaxchain.testnet [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=2192
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=43
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+BRIDGE_INIT_FINALITY=1
+# TODO: This exists. Is this what we should use?
+BRIDGE_INIT_WETH=0x4200000000000000000000000000000000000006
```
Comment (evan-gray) [inline ethereum/env/.env.snaxchain.testnet:21]: yes, this is the wrapped custom gas address for OP Stack chians. https://docs.optimism.io/stack/protocol/features/custom-gas-token#what-is-the-wrapped-erc-20-gas-token

**Review by evan-gray — APPROVED** [review]

File: sdk/js/src/utils/consts.ts (hunk 1 above)
Comment (bruce-riley) [reply sdk/js/src/utils/consts.ts:41]: I assume we still want to support the old sdk.

I am currently investigate how to (who should??) update sdk v2. That will obviously be a separate PR anyway, so I'm resolving this issue.

**Review by panoel — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]

**Pull request merged**